web/client/components/map/openlayers/Map.jsx

```diff
@@ -146,7 +147,7 @@ class OpenlayersMap extends React.Component {
             controls: controls,
             interactions: interactions,
             maxTilesLoading: Infinity,
-            target: `${this.props.id}`,
+            target: this.props.document?.getElementById(this.props.id) || `${this.props.id}`,
```

Comment:
what about this?
```suggestion
            target: this.getDocument()?.getElementById(this.props.id)`,
```
if you agre please update also other maps

Reply:
done, but optional chaining is not useful in this case because getDocument always returns something (document is always defined in a web page)

Reply:
right,